domains/doing.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/doing1024/note",
+    "owner": {
+        "username": "doing1024",
+        "email": "doing1024@outlook.it",
+        "twitter": "sorry,i don't have twitter"
```

Comment:
If you don't have twitter you can just remove this section (If you are gonna remove this section don't forget to remove the comma at twitter line)